docs/setup_ui.md

```diff
@@ -26,7 +26,7 @@ in your runtime path:
 
 ```shell
 $ node --version
-v18.19.0
+v22.0.0
 $ npm -v
 10.3.0
```

Comment:
with the latest image, I get the following versions. let's use this as an example. specifically that npm looks old in our guide:

$ node -v
v22.15.1
$ npm -v
10.9.2